soccer/Processor.cpp

```diff
@@ -296,10 +299,19 @@ void Processor::run() {
             log->CopyFrom(packet->wrapper);
 
             curStatus.lastVisionTime = packet->receivedTime;
+
+            if (packet->wrapper.has_geometry()) {
+                // DEMO: Test out field sizes
+                const SSL_GeometryFieldSize fieldSize =
+                    packet->wrapper.geometry().field();
+                // FIXME - Account for network latency
```

Comment:
Ok gotcha.  Yeah if you want to just delete it, that would be fine.  Latency calculation has been a long-running todo item #166 , #293